aladdin.sh

```diff
@@ -284,6 +284,14 @@ function prepare_docker_subcommands() {
     fi
 }
 
+function synchronize_datetime()
+{
+  # When minikube wakes up from sleep, date or time could be out of sync.
+  # Take date + time from host and set it on minikube.
+  echo "Synchronizing date and time on minikube with the host"
+  ssh -i ~/.minikube/machines/minikube/id_rsa docker@$(minikube ip) "docker run --rm --privileged --pid=host alpine nsenter -t 1 -m -u -n -i date -u $(date -u +%m%d%H%M%Y)"
```

Comment:
I think this might need some path normalizing depending on the OS

Reply:
actually, I think using `minikube ssh` might be better than plain `ssh`

Reply:
we can run the `docker run` command by itself from within aladdin-container.sh

Reply:
As per PTR's suggestion: skipped docker step. Just minikube ssh and set date.